lisa/sut_orchestrator/azure/common.py

```diff
@@ -243,13 +429,121 @@ def __hash__(self) -> int:
             f"{self.image_version}"
         )
 
+    def _get_info(self, platform: "AzurePlatform") -> Optional[Dict[str, Any]]:
+        self.resolve_version(platform)
+        sig_info = platform._get_sig(self)
+        return _get_image_tags(sig_info)
+
+    def _parse_info(self, raw_features: Dict[str, Any]) -> None:
+        # CPU Architecture
+        arch = raw_features.get("architecture")
+        if arch == "arm64":
```

Comment:
Some logic are similar among marketplace, sig and community image. Create a shared method in AzureImageSchema to reduce duplicate code.